Update API and secret key management doc (#675)
amplitude/amplitude-docs: content/collections/account-management/en/manage-your-api-keys-and-secret-keys.md

You should review the [keys and tokens documentation](https://amplitude.com/docs/apis/keys-and-tokens) before you make any changes to your keys.
{{/partial:admonition}}

Amplitude’s self-service key management page helps you create, revoke, or delete both API keys and secret keys at the project level. **All changes are permanent**.

You can: 
* Create and name multiple API keys and secret keys
* View a log of the creator, and the last action taken on the key
* Disable API access

{{partial:admonition type='note'}}

> Once you revoke an API key, you can't undo it. However, you will still be able to see the key value.  {{/partial:admonition}}

Reviewer: ```suggestion
Once you revoke an API key, you can't undo it. However, you can still view the key value.  {{/partial:admonition}}
```
